tests/unittests/OperatorTest.cpp

```diff
@@ -1556,14 +1556,17 @@ TEST_P(Operator, TestQuantizedRescaleSequence) {
   auto T4 = mod_.uniqueType(ElemKind::Int8QTy, {len}, 0.1, 7);
   auto T5 = mod_.uniqueType(ElemKind::Int8QTy, {len}, 0.3, -3);
 
+  auto T6 = mod_.uniqueType(ElemKind::FloatTy, {len});
```

Comment:
why do you need to change this test?

Reply:
ahh, you are right.  I was confused why it worked before my implementation, then i realized that previously it might just pass float value to the kernel

Reply:
can you check why "RescaleNode" test does not fail (in the operatorTest).
Please, fix that and enable only on CPUandInterpreter after the fix.

Reply:
sure

Reply:
it seems that all the rescale nodes got optimized out before running